tests/unit/dom.ts

```diff
@@ -488,5 +489,47 @@ registerSuite({
 			assert.strictEqual(parent.children.length, 0);
 			assert.isNull(node.parentNode);
 		}
+	},
+
+	create: {
+		'correct element created'() {
+			let tagType = 'div';
+			let result = dom.create(tagType);
+			assert.strictEqual(result.nodeName.toLowerCase(), tagType);
+		},
+
+		'attributes and properties are set on element'() {
+			let args: CreateArgs = { foo: 'bar', baz: 'buz', attributes: { 'className': 'test' } },
+				element = dom.create('div', args);
+
+			assert.strictEqual((<any> element).foo, 'bar');
+			assert.strictEqual((<any> element).baz, 'buz');
+			assert.strictEqual(element.getAttribute('className'), 'test');
+		},
+
+		'String children are appended as Text nodes to element'() {
+			let element = dom.create('div', null, ['test']);
+			assert.strictEqual(element.childNodes.length, 1);
+			assert.strictEqual((<any> element.firstChild).nodeName.toLowerCase(), '#text');
+		},
+
+		'Node children are appended to element'() {
```

Comment:
This test needs to add at least 2 children to actually verify that they're indeed being appended.